jekyll/_cci2/gitlab-trigger-options.adoc

```diff
@@ -10,8 +10,57 @@ contentTags:
 :icons: font
 :experimental:
 
-GitLab **Trigger filters** allow you to determine when a trigger should initiate a build based on the parameters provided by Gitlab’s webhook. CircleCI provides some common options, for example, only build on merge requests, but you can also build your own rules using the custom filter option. For example, a custom filter would allow you to only build on a specific branch or user.
+GitLab **Trigger filters** allow you to determine when a trigger should start a build based on the parameters provided by GitLab's webhook.
+
+CircleCI provides some common options, for example, only build on merge requests, but you can also build your own rules using the custom filter option. For example, a custom filter would allow you to only build on a specific branch or user.
```

Comment:
I suggest moving this sentence to after the bullet list lines 21 and 22. That way we keep a very succinct opening and make the intro focus on the page subject